docs/Install.md

```diff
@@ -7,59 +7,45 @@
 If you want to customize the installation of the Splunk Operator, download a copy of the installation YAML locally, and open it in your favorite editor.
 
 ```
-wget -O splunk-operator-cluster.yaml https://github.com/splunk/splunk-operator/releases/download/1.1.0/splunk-operator-cluster.yaml
+wget -O splunk-operator-cluster.yaml https://github.com/splunk/splunk-operator/releases/download/2.0.0/splunk-operator-cluster.yaml
 ```
 
 ## Default Installation
 
-By default operator will be installed in `splunk-operator` namespace and will watch all the namespaces of your cluster for splunk enterprise custom resources
+By default operator will be installed in `splunk-operator` namespace and based on the file used it can be installed cluster-wide or namespace scoped. By installing `splunk-operator-cluster.yaml` Operator will watch all the namespaces of your cluster for splunk enterprise custom resources
```

Comment:
we can mention file names here

Reply:
probably, we can highlight the config item that is making the difference here with respect to the namespace selection.

Reply:
this is tricky, we may have splunk-operator creation at one place but its also used in other resources when the creation happens.

Reply:
I have updated documentation , user can change the namespace by editing the manifest file `splunk-operator-namespace.yaml` or `splunk-operator-cluster.yaml` whichever they are using.